### ISSUE TITLE
License

### ISSUE THREAD

**Issue by lucko** [NickImpact/GTS#21]
Hey!

I've noticed that a considerable portion of the code in the `storage` package has been copied (verbatim in some cases) from my project [LuckPerms](https://github.com/lucko/LuckPerms).

https://github.com/lucko/LuckPerms/tree/6602b23f091140e1677b1a2ef93983a969ba6de4/common/src/main/java/me/lucko/luckperms/common/storage

https://github.com/NickImpact/GTS/tree/037a871f32c4c1568c7381844d52c84ed6bb92ba/src/main/java/com/nickimpact/gts/storage

It's awesome that you've been able to reuse the code in some form, and have found some use for it, but I'd appreciate it if you followed the license LuckPerms is released under.

I chose to [license the project under MIT](https://github.com/lucko/LuckPerms/blob/master/LICENSE.txt) (regarded as one of the most permissive open source software licenses) so this sort of use could happen - but hope to see the minor requirements being followed.

The license states:

> The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software

or in a simpler form

> You must preserve copyright and license notices

These notices and headers have been stripped from your copy - and you have accredited *yourself* as the author in the JavaDoc headers - which seems rather dishonest.

I have no intention of taking this any further - but am kindly asking that you correct this by reinstating the license headers.

https://github.com/lucko/LuckPerms/blob/6602b23f091140e1677b1a2ef93983a969ba6de4/common/src/main/java/me/lucko/luckperms/common/storage/Storage.java#L1-L24

Thanks :)

**Comment by NickImpact** [comment]
Should be all set in 985f28e, sorry for the troubles again!

**Comment by NickImpact** [comment]
Yeah, was actually going to surprise you with those being documented, just didn't have time in the original commit :P

**Comment by NickImpact** [comment]
Documented with ff40a0d, let me know if there are any outlying cases I missed that you notice. I'll be doing a review of the code tomorrow to ensure there aren't anymore outliers!
